rust/lance-encoding/src/buffer.rs

```diff
@@ -225,8 +222,8 @@ impl LanceBuffer {
     /// Reinterprets a LanceBuffer into a Vec<T>
     ///
     /// If the underlying buffer is not properly aligned, this will involve a copy of the data
+    #[cfg(target_endian = "little")]
```

Comment:
I would prefer to have this `enforcing machine endian` logic somewhere more public, for example a `build.rs` and/or `Cargo.toml` in project root.

Reply:
Alright, I put a directive in `lib.rs`.  I prefer to avoid fiddling with `build.rs` if possible.